### ISSUE TITLE
Feynman Path Integral

### ISSUE THREAD

**Issue by Roger-luo** [JuliaQuantum/QuDynamics.jl#28]
is there a Monte Carlo Method for Feynman Path Integral?I didn't find one in the doc

**Comment by acroy** [comment]
Not yet. What specifically do you have in mind?

**Comment by Roger-luo** [comment]
not any good idea yet, but I just think a simplest Monte Carlo method for Feynman Path Integral is necessary.cause sometimes people may use path integral to calculate a wave function by force. BTW the QuBase.jl or some other packages may need a Propagator type for path integral?

**Comment by acroy** [comment]
Yes, it would definitely be good to have some PIMC propagator. One possible problem could be that at the moment everything in QuBase/QuDynamics is about discrete bases? If you have a good reference it would be great of you could share it. The general workflow of adding a new propagator type is described in [this wiki article](https://github.com/JuliaQuantum/QuDynamics.jl/wiki/Adding-a-new-solver-to-QuDynamics-!).

**Comment by Roger-luo** [comment]
Sure,I would try to write one accroding to the wiki article
